src/components/PartyProfileBar/index.js

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+
+const rowStyle = {
+  display: "flex",
+  flexDirection: "row",
+  flex: 1,
+}
+
+const columnStyle = {
+  display: "flex",
+  flexDirection: "column",
+}
+
+const partyProfileBarStyle = {
+  ...columnStyle,
+  margin: "20px 0",
+}
+
+const labelRowStyle = {
+  ...rowStyle,
+  marginBottom: 3,
+}
+
+const labelStyle = {
+  ...columnStyle,
+  justifyContent: "flex-end",
+  fontFamily: "var(--ff-title)",
+  fontSize: 14,
+}
+
+const chartStyle = {
+  ...rowStyle,
+  border: "1px solid var(--cl-black)",
+  width: "100%",
+}
+
+const chartCellStyle = {
+  ...columnStyle,
+  fontSize: 10,
+  padding: "10px 2px",
+  borderRight: "inherit",
+  "&:last-child": {
+    borderRightWidth: 0,
+  },
+}
+
+const PartyProfileBar = ({ dataset }) => {
+  return (
+    <div css={partyProfileBarStyle}>
+      <div css={labelRowStyle}>
+        {dataset.map(({ label, value }) => (
+          <div key={label} css={{ ...labelStyle, width: `${value}%` }}>
+            {label}
+          </div>
+        ))}
+      </div>
+      <div css={chartStyle}>
+        {dataset.map(({ label, value, color }) => (
+          <div
+            key={label}
+            css={{
+              ...chartCellStyle,
+              width: `${value}%`,
```

Comment:
Regarding the use of `width`, I suggest you to separate it as `style={{ width: ... }}`.

The reason is that when using `css={{}}` it compiles to `class="..."`, while `style={{ }}` compiles to, surprisingly, `style="..."`. Try switching between these two and inspect it for yourself.

That being said, width attribute is kinda dynamic here depending on data, so using "style" is better I think.

Reply:
Ohhhh this is a very good point. Never think about that before.